### PR TITLE
Added RGB format document

### PR DESCRIPTION
**Pull request by TurkeyMan**

### CODE REVIEW

**Comment by p0nce** [comment]
https://wiki.multimedia.cx/index.php/YUV4MPEG2

I think only these five subsampling scheme (provides chroma sample "sitting" + subsampling amout) need to be considered, if subsampling has to be considered.
```
    '420jpeg' = 4:2:0 with biaxially-displaced chroma planes
    '420paldv' = 4:2:0 with vertically-displaced chroma planes
    '420' = 4:2:0 with coincident chroma planes
    '422' = 4:2:2
    '444' = 4:4:4
```

**Comment by TurkeyMan** [comment]
Bit-plane formats aren't a concern here, those are composited from multiple images buffers with a compositing algorithm.
My interest here is strictly the expression of interleaved data formats... RG_BG is one such format where you have to sample 2 pixels to read colour for either of them; that's a form of block compression.
The general YUV family of colour spaces would be expressed separately with focused API.

File: rgb-format.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+- `e` - shared exponent
+- `x` - unused bits
+
+Component names are lower-case. (or should they be upper case? they could be case-insensitive, but then we halve the namespace for future expansion)
+
+### `format` [optional]:  
+Description of the format for each channel, underscore separated for each component
+Eg: `rgb_8_8_8`, `bgra_10_10_10_2`, `rgbe_9_9_9_5`, `rgba_s6_f11_4.8_u3`  
+Encoding options:
+- Omitted: 8 bit normalised integer for each compoent
```
Comment (drug007) [inline rgb-format.md]: typo in `component`

Comment (TurkeyMan) [reply rgb-format.md]: Thanks!

Comment (TurkeyMan) [reply rgb-format.md]: https://github.com/DlangGraphicsWG/documents/pull/16

**Pull request merged**